sample_factory/runner/runs/mujoco_all_envs.py

```diff
@@ -0,0 +1,16 @@
+from sample_factory.runner.run_description import RunDescription, Experiment, ParamGrid
+
+_params = ParamGrid([
+    ('env', ['mujoco_ant', 'mujoco_halfcheetah', 'mujoco_hopper', 'mujoco_humanoid', 'mujoco_doublependulum', 'mujoco_pendulum', 'mujoco_reacher', 'mujoco_swimmer', 'mujoco_walker']),
+])
+
+_experiments = [
+    Experiment(
+        'mujoco_all_envs',
+        'python -m sample_factory_examples.mujoco_examples.train_mujoco --algo=APPO',
+        _params.generate_params(randomize=False),
+    ),
+]
+
+
+RUN_DESCRIPTION = RunDescription('mujoco_all_envs', experiments=_experiments)
```

Comment:
This is missing multiple seeds per run. Ideally we need to evaluate like 5-10 seeds on each env, at least for the final result.